### PR TITLE
Add Vision Subsystem

### PR DESCRIPTION
**Pull request by LQ1234**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/Vision.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        INTAKE,
+        SHOOTER,
+        HUBFINDER,
+    }
+
+    private static final double[] EMPTY_TARGET_INFO = new double[] {0.0,0.0,0.0,0.0,0.0,0.0,0.0};
+
+
+    private Relay m_spike;
+    private DriveTrain m_driveTrain;
```
Comment (prensing) [inline src/main/java/frc/robot/subsystems/Vision.java:19]: Why does the Vision system need the Drivetrain?

**Review by prensing — CHANGES_REQUESTED** [review]
I don't think we want the drivetrain here, at least not yet. Add it when it is needed.

File: src/main/java/frc/robot/subsystems/Vision.java (hunk 1 above)
Comment (LQ1234) [reply src/main/java/frc/robot/subsystems/Vision.java:19]: For simulation

Comment (prensing) [reply src/main/java/frc/robot/subsystems/Vision.java:19]: Then you should add that as a comment.

**Review by coachFitz0807 — APPROVED** [review]

**Review by prensing — APPROVED** [review]

**Pull request merged**